etc/config.sample.toml

```diff
@@ -1,125 +1,80 @@
 # Welcome to the InfluxDB configuration file.
 
-# If hostname (on the OS) doesn't return a name that can be resolved by
-# the other systems in the cluster, you'll have to set the hostname to an
-# IP or something that can be resolved here.
-# hostname = ""
+[meta]
+  dir = "/var/opt/influxdb/meta"
+  hostname = "localhost"
+  bind-address = ":8088"
+  retention-autocreate = true
+  election-timeout = "1s"
+  heartbeat-timeout = "1s"
+  leader-lease-timeout = "500ms"
+  commit-timeout = "50ms"
 
-bind-address = "0.0.0.0"
-
-# The default cluster and API port
-port = 8086
-
-# Once every 24 hours InfluxDB will report anonymous data to m.influxdb.com
-# The data includes raft id (random 8 bytes), os, arch and version
-# We don't track ip addresses of servers reporting. This is only used
-# to track the number of instances running and the versions, which
-# is very helpful for us.
-# Change this option to true to disable reporting.
-reporting-disabled = false
+[data]
+  dir = "/var/opt/influxdb/data"
+  retention-auto-create = true
+  retention-check-enabled = true
+  retention-check-period = "10m0s"
+  retention-create-period = "45m0s"
 
-# Controls settings for initial start-up. Once a node is successfully started,
-# these settings are ignored.  If a node is started with the -join flag,
-# these settings are ignored.
-[initialization]
-join-urls = "" # Comma-delimited URLs, in the form http://host:port, for joining another cluster.
+[cluster]
+  shard-writer-timeout = "5s"
 
-# Control authentication
-# If not set authetication is DISABLED. Be sure to explicitly set this flag to
-# true if you want authentication.
-[authentication]
-enabled = false
+[retention]
+  enabled = true
```

Comment:
Something is wrong here -- I think the retention control in the `data` section is obsolete. I -- or you -- would need to check the code.